### PR TITLE
Add namespace labels option to assist with the AWS App Mesh requirement

### PR DESCRIPTION
**Pull request by thegippygeek**
*Issue #, if available:* 293

*Description of changes:* Added namespace labels to the teams namespace  

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: lib/teams/index.ts [hunk 1]
```diff
@@ -28,6 +28,11 @@ export class TeamProps {
      */
     readonly namespaceAnnotations? : { [key: string]: any; } = { "argocd.argoproj.io/sync-wave": "-1" };
 
+    /**
+     * Labels such as necessary for AWS AppMesh 
+     */
+    readonly namespaceLabels? : { [key: string]: any; } = {"appmesh.k8s.aws/sidecarInjectorWebhook": "enabled"};
```
Comment (shapirov103) [inline lib/teams/index.ts:34]: Let's not assign this default to the namespace. We should have labels, but the content should be provided by the customers when they define their teams.

**Review by shapirov103 — COMMENTED** [review]
Thank you @thegippygeek for your contribution. It is definitely a needed feature. I left one comment, we can merge after.

File: lib/teams/index.ts (hunk 1 above)
Comment (thegippygeek) [reply lib/teams/index.ts]: Yes 100% agree, removed as suggested.

**Review by shapirov103 — APPROVED** [review]
LGTM

**Pull request merged**